automation_tools/__init__.py

```diff
@@ -1594,6 +1697,20 @@ def product_install(distribution, create_vm=False, certificate_url=None,
             sat_version=satellite_version,
             host=host
         )
+    if os.environ.get('IDM_EXTERNAL_AUTH') or os.environ.get('IDM_REALM'):
```

Comment:
Just realized even for `false` this would be `true`. will be fixing it shortly.
Same for below references.